pom.xml

```diff
@@ -158,7 +159,7 @@
         <ranger.version>1.2.0</ranger.version>
         <plexus.version>3.3.0</plexus.version>
         <quartz.version>2.3.2</quartz.version>
-        <lucene-analyzers-common.version>7.0.0</lucene-analyzers-common.version>
+        <lucene-analyzers-common.version>7.7.2</lucene-analyzers-common.version>
```

Comment:
See #1106 - this doesn't take us to 7.8 but hopefully will address some of the issues

I'll suggest running an owasp check (-DCVE on the build) - I can pull and check - though I think your proposed change is first step in any case

Reply:
owasp checks are failing. Will review in nexus-iq reports after merge